.github/workflows/release.yml

```diff
@@ -1,26 +1,116 @@
 name: Release
 
 on:
-  push:
-    tags:
-      - '[0-9]+.[0-9]+.[0-9]+'
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        type: choice
+        options:
+          - minor
+          - patch
+        default: minor
+        required: true
+        description: 'patch: v0.0.X, minor: v0.X.0'
```

Comment:
Please switch patch + minor fields the description to: `minor: v0.X.0, patch: v0.0.X`